index.html

```diff
@@ -2,12 +2,77 @@
 <html>
 <head>
 <link href="style.css" rel="stylesheet" type="text/css">
+<script src="https://kit.fontawesome.com/173db1d6d6.js" crossorigin="anonymous"></script>
 <title>Portafolio</title>
 </head>
 <body>
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+    
 
+    <div class="wrapper">
+       
+        <div class="container">
+            <div  class="hour">      9:31 </div>
+            <div  class="indicators"> 
+                <img src="./images/status_mobile.png">
+            </div>
+
+        </div>
+    
+        <nav class="nav">            
+            <div class="name"> Edahí  </div>      
+            <div class="hamburger" ><img  src="./images/Hamburger.png">  </div>          
+        </nav>
+    
+        
+        
+        <h1 class="info">Hey There. <br>
+            I’m Edahí</h1>
+
+        <p class="intro">I’am a Software Developer</p>   
+    
+        <p> I can help you build a product ,
+            feature or website Look through some of my work
+            and experience! If you like what you see and have
+            a project you need coded,  don’t hestiate to contact me. 
+        </p>        
+            
+    
+        <div class="social-group">
+            <ul>
+                <li class="social">
+                    <a href="http://github.com/edahigure">
+                        <img src="./images/git.png"> </a>
+                </li>
+                <li class="social">
+                    <a href="https://linkedin.com/in/edahigure">                    
+                    <img src="./images/likedin.png">
+                </li>
+
+                <li class="social">
+                    <a href="#">
+                    <img src="./images/angelist.png"> </a>                       
+                </li>                
+
+                <li class="social">
+                    <a href="https://twitter.com/edahigure">
+                        <img src="./images/twiter.png"> </a> 
+                </li>    
+                
+                <li class="social">        
+                    <a href="#">
+                        <img src="./images/m.png"> </a>
+                </li>
+                <li class="image"></li>                
+            </ul>                            
+        </div>       
```

Comment:
- Kindly wrap this block of code inside a `section` tag. This block of code represents a generic standalone section of the document so it has to be inside a `section` tag. :+1: :smiley: :smiley: